source/infrastructure/pipeline/e2e-tests/basic-ec2-start-stop.test.resources.ts

```diff
@@ -0,0 +1,74 @@
+/*****************************************************************************
+ *  Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.   *
+ *                                                                            *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may   *
+ *  not use this file except in compliance with the License. A copy of the    *
+ *  License is located at                                                     *
+ *                                                                            *
+ *      http://www.apache.org/licenses/LICENSE-2.0                            *
+ *                                                                            *
+ *  or in the 'license' file accompanying this file. This file is distributed *
+ *  on an 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,        *
+ *  express or implied. See the License for the specific language governing   *
+ *  permissions and limitations under the License.                            *
+ *****************************************************************************/
+import * as ec2 from "aws-cdk-lib/aws-ec2";
+import * as cdk from "aws-cdk-lib";
+
+import {Construct} from "constructs";
+import {NagSuppressions} from "cdk-nag";
+import {TestResourceProvider} from "./index";
+
+
+
+const envKeys = {
+  ec2InstanceId: 'basic_start_stop_instance_id',
+}
```

Comment:
I believe the type is inferred automatically here, and this particular object is little more than a private string reference that should never be used by other modules. I just built it this way because it made more semantic sense than `EC2_INSTACE_ID_ENV_KEY`